jax/_src/ad_util.py

```diff
@@ -95,7 +95,7 @@ def _stop_gradient_impl(x: T) -> T:
 
 
 class SymbolicZero:
-  def __init__(self, aval: core.AbstractValue) -> None:
+  def __init__(self, aval: core.AbstractValue):
```

Comment:
Quoting [PEP 484](https://peps.python.org/pep-0484/): "Note that the return type of __init__ ought to be annotated with `-> None`" 😛

Reply:
(I think it's optional these days, I just like it because `__init__` is just a regular method!)